docs/distributing-your-plugin.md

```diff
@@ -50,7 +50,7 @@ These screenshots are stored [in git: CustomAlerts/screenshots](https://github.c
 <a name="plugin-json-required-fields"></a>
 The `plugin.json` file must contain the following information:
 
-- `name`: The plugin's name. It can only contain letters (a-z), numbers (0-9) and must start with a capital letter. The name cannot contain the words `"Piwik"`, `"Core"` or `"Analytics"`. A maximum of 40 characters are allowed.
+- `name`: The plugin's name. It can only contain letters (a-z), numbers (0-9) and must start with a capital letter. The name cannot contain the words `"Piwik"`, `"Core"` or `"Analytics"`. A maximum of 60 characters are allowed.
```

Comment:
This file doesn't seem to be version related. So simply changing the number might not be good.
@tsteur I'm not sure how that exactly works: Can we get that file version dependent by simply copying it to one or all version directories?

Reply:
We could copy it into a version specific folder eg `docs/4.x` and `docs/5.x`. As it's maybe unlikely someone will create a plugin which such a length over the next month for a soon outdated Matomo maybe it be fine to have not version controlled. Either way can be done though 👍